### ISSUE TITLE
Video Access Settings and Privacy Restrictions

### ISSUE THREAD

**Issue by gbrn** [WWBN/AVideo#9757]
@DanielnetoDotCom For a few days now, I have been facing a problem regarding user access to certain videos.
I addressed this issue a long time ago in issue #7438.
But now I am being plagued by a problem, that all the videos are available and not only through login and password.
I thought it was a problem with my settings, but I was unable to reset the settings to access the videos, only logged-in users.
I also noticed that when accessing the video in "Edit" and then "Privacy" and selecting the user groups, they are blocked.

I still don't know what's happening, could you give me some insight into this problem?

[1.webm](https://github.com/user-attachments/assets/f42d116f-9eb9-4af7-b264-fb60686b4b92)

**Comment by DanielnetoDotCom** [comment]
your video does not show anything

**Comment by DanielnetoDotCom** [comment]
Check your JS console for clues

**Comment by gbrn** [comment]
@DanielnetoDotCom First of all, Happy New Year Danny!!!

Regarding the problem reported previously... the problem with the buttons and JS is the least of it.

See below that I am having serious problems with accessibility to videos. In the example below, I show a film shown on BIG VIDEO that is "Unpublished".

This problem has only been occurring to me recently. I don't know what is happening. I need to know if there is or has been any additional configuration included for this to happen.

[CNETV.webm](https://github.com/user-attachments/assets/7eeb17c1-ac83-4470-bcff-bc7193d7b2e3)

**Comment by DanielnetoDotCom** [comment]
Regarding the Unpublished, are you using admin user?

If the Unpublished plays it would play only to admin. 

can you try with a non admin user?

but I guess it should not play even to admin

**Comment by gbrn** [comment]
I also agree that it should be blocked for the Administrator, as it has always been.
The issue is that all videos are released to the general public, without any blocking. Without requiring any login.
Can you guide me, what is the configuration in the plugin that I can test to try to remedy this problem?

**Comment by DanielnetoDotCom** [comment]
Unpublished is when you select a date on encoder


https://github.com/WWBN/AVideo/wiki/Create-non-public-videos-and-deal-with-user-groups

**Comment by gbrn** [comment]
Apparently the problem started after I enabled the **Subscription Plugin** and activated the _"allowFreePlayWithAds"_ option.
I'm still investigating.

![image](https://github.com/user-attachments/assets/6f077c2a-52a3-473f-8725-cdef6b56b6ef)

**Comment by gbrn** [comment]
Do some tests, Danny... with "allowFreePlayWithAds" enabled, the system allows free access to any video.

Even disabling the Subscription plugin completely restores the previous access control configuration as it should be.

Could you check this for us?

**Comment by DanielnetoDotCom** [comment]
Just confirming, when you select allowFreePlayWithAds the Unpublish videos start to play?

**Comment by gbrn** [comment]
@DanielnetoDotCom Danny, I think we have two distinct problems here.

1- Using the ADMIN account, I verified that a video marked as _Unpublished_ is being displayed on BIG VIDEO.
But it does not appear in the search.
In other words, regardless of whether `allowFreePlayWithAd` is enabled, the _Unpublished_ video is displayed prominently on BIG VIDEO.

![image](https://github.com/user-attachments/assets/d1f4ba27-ad60-4043-9574-2bc31f2df321)


[CNETV1.webm](https://github.com/user-attachments/assets/a2f111f7-1f45-4945-bbc9-64095cdafbcc)

When I access this same video _Unpublished_, with a non-administrator user, the following message appears:
![image](https://github.com/user-attachments/assets/4b0d5d1f-f8ad-4447-a97a-79e1f36bac10)


2- But with the Plugin Subscription, and `allowFreePlayWithAd` disabled, the system asks for login as it should.

**Comment by DanielnetoDotCom** [comment]
Confirming again

1. Big video is showing Unpublished videos regardless of whether allowFreePlayWithAd is enabled
2. if allowFreePlayWithAd is enabled users can play Unpublished videos 

is that correct?

**Comment by gbrn** [comment]
> 1. I am checking the code, admin can see unpublished videos

Yes, access was done via ADMIN.

> 2. logged users can see their own unpublished videos

I didn't test other users.

I did the most recent update. Now I'll check it out because the BIG VIDEO is random.

**Comment by gbrn** [comment]
> I also noticed that when accessing the video in "Edit" and then "Privacy" and selecting the user groups, they are blocked.

https://private-user-images.githubusercontent.com/37880073/399381018-f42d116f-9eb9-4af7-b264-fb60686b4b92.webm?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzY1MTI1NDksIm5iZiI6MTczNjUxMjI0OSwicGF0aCI6Ii8zNzg4MDA3My8zOTkzODEwMTgtZjQyZDExNmYtOWViOS00YWY3LWIyNjQtZmI2MDY4NmI0YjkyLndlYm0_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMTEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDExMFQxMjMwNDlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jNzA4ZmFjM2YzM2FkMmY0Y2NmMGVkN2E1ZGYwNDY1NWRlZDljYWQ1YTk3MWUyYjBlMTcwOTkzYmMyNmU5ZDQwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.gmRBOnMhTxZM-8NG_1HCPzjbQzEkkAN_LCQT1vwoDCQ


It is important to note that the Privacy selections, "Groups that can view this video", are in black, with no option to select. This is because the category selected for the video is already defining the groups.
@DanielnetoDotCom If you can document this on the WIKI... thanks.
